### DISCUSSION TITLE
Get information Token

### DISCUSSION THREAD

**Discussion by erickximenes** [adonisjs/core#2682] [Help] [answered]
Is there any way to obtain the information of the Token generated at the moment of Login and create a middleware to verify this token.

**Comment by d4n1b** [comment, marked as answer]
Hi @ZaunSupremoXV ,
Is this not managed automatically by AdonisJS auth module? You can read the basic [here](https://docs.adonisjs.com/guides/auth/basic-auth-guard) & about the middleware [here](https://docs.adonisjs.com/guides/auth/middleware)

I hope it helps.

**Reply by asta-nguyen** [reply]
How can I receive user decode information from a token at any point in my code?
Excluding a token
